packages/reports/src/Constants/__tests__/data-table-constants.spec.tsx

```diff
@@ -0,0 +1,570 @@
+        currency,
+        onClickCancel,
+        onClickSell,
+        getPositionById,
+        server_time,
+    });
+
+    const typeColumn = columns[0];
+    const limitOrderColumn = columns[6];
+    const bidPriceColumn = columns[7];
+    const profitColumn = columns[8];
+    const actionColumn = columns[9];
+
+    const dummyRowObj = {
+        id: 'dummy_id',
+        contract_info: {
+            id: 'contract_info_id',
+            currency: 'USD',
+            profit: 200,
+            buy_price: 1000,
+            bid_price: 1200,
+            multiplier: 2,
+            limit_order: {
+                take_profit: { order_amount: 1500 },
+                stop_loss: { order_amount: 500 },
+            },
+            cancellation: { ask_price: 50 },
+            contract_id: 'contract_123',
+            underlying: '1HZ10V',
+        },
+        barrier: 1.2345,
+        is_sell_requested: false,
+        profit_loss: 50,
+    };
+    const cell_value = '1000';
+    const input_mocked_obj = {
+        row_obj: dummyRowObj,
+        cell_value,
+        is_footer: false,
+        is_vanilla: false,
+        is_turbos: false,
+        passthrough: {},
+    };
+
+    it('should render all columns correctly', () => {
+        columns.forEach(column => {
+            const { col_index } = column;
+            if (col_index) {
+                render(<div>{col_index}</div>);
+                expect(screen.getByText(col_index)).toBeInTheDocument();
+            }
+        });
+    });
+
+    it('should render cell content correctly for each column', () => {
+        columns.forEach(column => {
+            const { renderCellContent: content, col_index } = column;
+            if (content) {
+                //@ts-expect-error cell_value type should be React.ReactNode
+                const cellContent = content({ ...input_mocked_obj });
+                render(<div>{cellContent}</div>);
+
+                switch (col_index) {
+                    case 'type':
+                        expect(screen.getByText(dummyRowObj.contract_info.id)).toBeInTheDocument();
+                        break;
+                    case 'multiplier':
+                        expect(screen.getByText(`x${dummyRowObj.contract_info.multiplier}`)).toBeInTheDocument();
+                        break;
+                    case 'currency':
+                        expect(screen.getByText(dummyRowObj.contract_info.currency)).toBeInTheDocument();
+                        break;
+                    default:
+                        break;
+                }
+            }
+        });
+    });
+
+    it('should render "Total" for the "Type" column when is_footer is true', () => {
+        const { renderCellContent: content } = typeColumn;
+        //@ts-expect-error cell_value type should be React.ReactNode
+        const cellContent = content({
+            ...input_mocked_obj,
+            is_footer: true,
+        });
```

Comment:
```suggestion
        }) as JSX.Element;
```
should help against the error instead of the comment :) 
please check with the similar places in this file too🙏🏻